node_rate: 100 #Hz
codes_self: [1,2,3,4]
robot_names: "quad99 quad98 quad97 quad96"
robot_codes_n: [2,2,2,2]
robot_codes: [1,2,3,4,5,6,7,8]
marker_pos: [0.25,0.0,0.0,0.0,-0.25,0.0,-0.25,0.0,0.0,0.0,0.25,0.0]
lag: 0.0 #seconds
w_th: 2.0 # meters / second^2 (w_th and w_yaw are egomotion noise, the noise of the estimated robots is much higher and is hardcoded)
w_roll: 0.05 # rads
w_pitch: 0.05 # rads
w_change: 0.0010 # in change rate
w_yaw: 0.5 # radians / second
v_Z: 0.05 # meters
v_OF: 0.1 # meters / second
